ci-operator/jobs/openshift/cluster-samples-operator/openshift-cluster-samples-operator-master-presubmits.yaml

```diff
@@ -609,550 +609,6 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )images,?($|\s.*)
-  - agent: kubernetes
```

Comment:
you want to add equivalent okd-scos jobs here, by defining the test config in `openshift-cluster-samples-operator-master__okd-scos.yaml`